lisp/modules/rename_cues/rename_ui.py

```diff
@@ -0,0 +1,299 @@
+        self.upperButton.clicked.connect(self.onUpperButtonClicked)
+        self.layout().addWidget(self.upperButton, 5, 0)
+
+        self.removeNumButton = QPushButton()
+        self.removeNumButton.setText(translate('RenameCues', 'Remove Numbers'))
+        self.removeNumButton.clicked.connect(self.onRemoveNumButtonClicked)
+        self.layout().addWidget(self.removeNumButton, 3, 1)
+
+        self.addNumberingButton = QPushButton()
+        self.addNumberingButton.setText(translate('RenameCues', 'Add numbering'))
+        self.addNumberingButton.clicked.connect(self.onAddNumberingButtonClicked)
+        self.layout().addWidget(self.addNumberingButton, 4, 1)
+
+        self.resetButton = QPushButton()
+        self.resetButton.setText(translate('RenameCues', 'Reset'))
+        self.resetButton.clicked.connect(self.onResetButtonClicked)
+        self.layout().addWidget(self.resetButton, 5, 1)
+
+        # Spacer
+        self.layout().addItem(QSpacerItem(30, 0), 3, 2)
+
+        # Regex lines
+        self.outRegexLine = QLineEdit()
+        self.outRegexLine.setPlaceholderText(
+            translate('RenameCues', 'Rename all cue. () in regex below usable with $0, $1 ...'))
+        self.outRegexLine.setToolTip(
+            translate('RenameCues', 'What you type here is used to rename all cues.\n'
+                                    'The regex line below is used to match patterns, and to '
+                                    'capture parts of the text with parenthesis.\n'
+                                    'You can use the content of first () here with $0, the second with $1, etc..'))
+        self.outRegexLine.textChanged.connect(self.onOutRegexChanged)
+        self.layout().addWidget(self.outRegexLine, 3, 3)
+
+        self.regexLine = QLineEdit()
+        self.regexLine.setPlaceholderText(translate('RenameCues', 'Type your regex here :'))
+        self.regexLine.setToolTip(translate('RenameCues', 'Match patterns with standard python regex.\n'
+                                            'Exemple : [a-z]([0-9]+) will find a lower case character [a-z] '
+                                            'followed by one or more number.\n'
+                                            'Only the numbers will be captured for use with $0 above.\n'
+                                            'For more information, consult python documentation'))
+        self.regexLine.textChanged.connect(self.onRegexLineChanged)
+        self.layout().addWidget(self.regexLine, 4, 3)
+
+        # OK / Cancel buttons
+        self.dialogButtons = QDialogButtonBox()
+        self.dialogButtons.setStandardButtons(QDialogButtonBox.Ok |
+                                              QDialogButtonBox.Cancel)
+        self.layout().addWidget(self.dialogButtons, 6, 3)
+
+        self.dialogButtons.accepted.connect(self.accept)
+        self.dialogButtons.rejected.connect(self.reject)
+
+    def get_selected_cues(self):
```

Comment:
this name is confusing, usually `get` prefix are used for functions that return something from the object, you can use `load`, or move the logic of this function in the `__init__` so that is executed only one time.

Same thing for the function below "`get_all_cues`"